### ISSUE TITLE
fix a11y issues

### ISSUE THREAD

**Issue by sastels** [cds-snc/report-a-cybercrime#799]
from the a11y scanner:
```
Violations on page: /: 1
- ***"id":"page-has-heading-one","impact":"moderate","tags":["cat.semantics","best-practice"],"description":"Ensure that the page, or at least one of its frames contains a level-one heading","help":"Page must contain a level-one heading","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/page-has-heading-one?application=axe-puppeteer","nodes":[***"any":[],"all":[***"id":"page-has-heading-one","data":null,"relatedNodes":[],"impact":"moderate","message":"Page must have a level-one heading"***],"none":[],"impact":"moderate","html":"<html lang=\"en\" data-rh=\"lang\">","target":["html"],"failureSummary":"Fix all of the following:\n  Page must have a level-one heading"***]***

Violations on page: /p2/whathappened: 1
- ***"id":"color-contrast","impact":"serious","tags":["cat.color","wcag2aa","wcag143"],"description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds","help":"Elements must have sufficient color contrast","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axe-puppeteer","nodes":[***"any":[***"id":"color-contrast","data":***"fgColor":"#808080","bgColor":"#ffffff","contrastRatio":3.94,"fontSize":"14.3pt","fontWeight":"normal","expectedContrastRatio":"4.5:1"***,"relatedNodes":[],"impact":"serious","message":"Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***],"all":[],"none":[],"impact":"serious","html":"<span class=\"css-1rdyqc3 e1fxharu0\">Tell us about it in your own words.</span>","target":[".css-1rdyqc3"],"failureSummary":"Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***]***

Violations on page: /p2/scammerdetails: 2
- ***"id":"color-contrast","impact":"serious","tags":["cat.color","wcag2aa","wcag143"],"description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds","help":"Elements must have sufficient color contrast","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axe-puppeteer","nodes":[***"any":[***"id":"color-contrast","data":***"fgColor":"#808080","bgColor":"#ffffff","contrastRatio":3.94,"fontSize":"14.3pt","fontWeight":"normal","expectedContrastRatio":"4.5:1"***,"relatedNodes":[],"impact":"serious","message":"Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***],"all":[],"none":[],"impact":"serious","html":"<span class=\"css-1rdyqc3 e1fxharu0\">Remember to include any email addresses, phone numbers, or website links</span>","target":[".css-1rdyqc3"],"failureSummary":"Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***]***
- ***"id":"heading-order","impact":"moderate","tags":["cat.semantics","best-practice"],"description":"Ensures the order of headings is semantically correct","help":"Heading levels should only increase by one","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/heading-order?application=axe-puppeteer","nodes":[***"any":[***"id":"heading-order","data":3,"relatedNodes":[],"impact":"moderate","message":"Heading order invalid"***],"all":[],"none":[],"impact":"moderate","html":"<h3 class=\"css-1evt1v8 eoh9q0q0\">0 files attached</h3>","target":["h3"],"failureSummary":"Fix any of the following:\n  Heading order invalid"***]***

Violations on page: /p2/timeframe: 1
- ***"id":"color-contrast","impact":"serious","tags":["cat.color","wcag2aa","wcag143"],"description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds","help":"Elements must have sufficient color contrast","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axe-puppeteer","nodes":[***"any":[***"id":"color-contrast","data":***"fgColor":"#808080","bgColor":"#ffffff","contrastRatio":3.94,"fontSize":"14.3pt","fontWeight":"normal","expectedContrastRatio":"4.5:1"***,"relatedNodes":[],"impact":"serious","message":"Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***],"all":[],"none":[],"impact":"serious","html":"<span class=\"css-1rdyqc3 e1fxharu0\">For example: 2019-04-28</span>","target":["label[for=\"startDate\"] > .css-1rdyqc3.e1fxharu0"],"failureSummary":"Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***,***"any":[***"id":"color-contrast","data":***"fgColor":"#808080","bgColor":"#ffffff","contrastRatio":3.94,"fontSize":"14.3pt","fontWeight":"normal","expectedContrastRatio":"4.5:1"***,"relatedNodes":[],"impact":"serious","message":"Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***],"all":[],"none":[],"impact":"serious","html":"<span class=\"css-1rdyqc3 e1fxharu0\">For example: 2019-04-28</span>","target":["label[for=\"endDate\"] > .css-1rdyqc3.e1fxharu0"],"failureSummary":"Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***]***

Violations on page: /p2/impact: 3
- ***"id":"checkboxgroup","impact":"critical","tags":["cat.forms","best-practice"],"description":"Ensures related <input type=\"checkbox\"> elements have a group and that the group designation is consistent","help":"Checkbox inputs with the same name attribute value must be part of a group","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/checkboxgroup?application=axe-puppeteer","nodes":[***"any":[***"id":"group-labelledby","data":***"name":"howWereYouAffected","type":"checkbox"***,"relatedNodes":[],"impact":"critical","message":"Elements with the name \"howWereYouAffected\" do not all have both a shared label, and a unique label, referenced through aria-labelledby"***,***"id":"fieldset","data":***"name":"howWereYouAffected","type":"checkbox","failureCode":"no-legend"***,"relatedNodes":[***"html":"<fieldset class=\"css-vtzyb eemyvxu1\">","target":["fieldset"]***],"impact":"critical","message":"Fieldset does not have a legend as its first child"***],"all":[],"none":[],"impact":"critical","html":"<input type=\"checkbox\" name=\"howWereYouAffected\" value=\"Device or account hacked\" meta=\"[object Object]\" checkscale=\"1\" id=\"id-18\" class=\"css-1skmiry e8tni2c0\">","target":["#id-18"],"failureSummary":"Fix any of the following:\n  Elements with the name \"howWereYouAffected\" do not all have both a shared label, and a unique label, referenced through aria-labelledby\n  Fieldset does not have a legend as its first child"***]***
- ***"id":"color-contrast","impact":"serious","tags":["cat.color","wcag2aa","wcag143"],"description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds","help":"Elements must have sufficient color contrast","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=axe-puppeteer","nodes":[***"any":[***"id":"color-contrast","data":***"fgColor":"#808080","bgColor":"#ffffff","contrastRatio":3.94,"fontSize":"14.3pt","fontWeight":"normal","expectedContrastRatio":"4.5:1"***,"relatedNodes":[],"impact":"serious","message":"Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***],"all":[],"none":[],"impact":"serious","html":"<span class=\"css-1rdyqc3 e1fxharu0\">For example: the amount of money, the information taken, what else was affected</span>","target":[".css-1rdyqc3"],"failureSummary":"Fix any of the following:\n  Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 14.3pt, font weight: normal). Expected contrast ratio of 4.5:1"***]***
- ***"id":"label","impact":"critical","tags":["cat.forms","wcag2a","wcag332","wcag131","section508","section508.22.n"],"description":"Ensures every form element has a label","help":"Form elements must have labels","helpUrl":"https://dequeuniversity.com/rules/axe/3.2/label?application=axe-puppeteer","nodes":[***"any":[***"id":"aria-label","data":null,"relatedNodes":[],"impact":"serious","message":"aria-label attribute does not exist or is empty"***,***"id":"aria-labelledby","data":null,"relatedNodes":[],"impact":"serious","message":"aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty"***,***"id":"implicit-label","data":null,"relatedNodes":[],"impact":"critical","message":"Form element does not have an implicit (wrapped) <label>"***,***"id":"explicit-label","data":null,"relatedNodes":[],"impact":"critical","message":"Form element does not have an explicit <label>"***,***"id":"non-empty-title","data":null,"relatedNodes":[],"impact":"serious","message":"Element has no title attribute or the title attribute is empty"***],"all":[],"none":[],"impact":"critical","html":"<textarea name=\"damage\" meta=\"[object Object]\" id=\"damage\" class=\"css-1brbsko e1vd0c6m0\"></textarea>","target":["#damage"],"failureSummary":"Fix any of the following:\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Form element does not have an implicit (wrapped) <label>\n  Form element does not have an explicit <label>\n  Element has no title attribute or the title attribute is empty"***]***
```
